php/commands/term.php

```diff
@@ -417,6 +417,36 @@ public function url( $args ) {
 		}
 	}
 
+	/**
+	 * Recount the term_taxonomy count for one or more taxonomies.
+	 *
+	 * <taxonomy>...
+	 * : One or more taxonomies to update
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     wp term recount category post_tag
+	 *
+	 *     wp taxonomy list --field=name | xargs wp term recount
+	 */
+	public function recount( $args ) {
+		foreach( $args as $taxonomy ) {
+
+			if ( ! taxonomy_exists( $taxonomy ) ) {
+				WP_CLI::warning( sprintf( "Taxonomy %s does not exist.", $taxonomy ) );
+			} else {
+
+				$terms = get_terms( $taxonomy, array( 'hide_empty' => false ) );
```

Comment:
What are the pros and cons of recounting terms for an entire taxonomy, vs. specifying term ids individually?